aws_marketplace/using_model_packages/preprocessing-audio-data-using-a-machine-learning-model/preprocessing-audio-data-using-a-machine-learning-model.ipynb

```diff
@@ -0,0 +1,868 @@
+{
```

Comment:
<p>Using s3 and S3. Stick with S3.</p><p>You have a ready to use S3 resource, so why don't you create the output bucket in the code? Seems like an error-prone spot that you don't setup the output folder in the code.</p><p> </p><p>Never just link to "here" as this doesn't help folks with accessibility readers. Link to "AWS Marketplace offering".</p><p> </p><p>1hr --&gt; one hour</p><p> </p><p>Please --&gt; You may</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3061/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>